tomcat/config.sls

```diff
@@ -28,6 +28,17 @@ tomcat tomcat_conf:
     - watch_in:
       - service: tomcat package installed and service running
 
+{% if tomcat.catalina_tmpdir|default %}
```

Comment:
Prefer to be explicit, to avoid future misunderstandings:

```suggestion
{% if tomcat.catalina_tmpdir|d('') %}
```

Ref: https://jinja.palletsprojects.com/en/2.11.x/templates/#default